docs/overview/app-container.md

```diff
@@ -69,6 +69,16 @@ export default container;
 
 The reason why we created the `AppContainer` class is to reduce the complexity of how the container is constructed using Inversify directly, also was to provide a way to register modules without too much extra configuration.
 
+:::note
```

Comment:
This can be removed as well, as everything is done using modules now

Reply:
You say about this alert?

```
:::note
InversifyJS contains a helper function called `buildProviderModule()` that can be used to build a module that automatically registers all providers and controllers in a given directory.

The function takes a string argument that represents the path to the directory containing the provider and controller classes, and returns a module that can be registered in the InversifyJS container. The module will automatically register all provider and controller classes in the directory and its subdirectories.

This is a useful feature when building large applications with many providers and controllers, as it allows you to easily register them without having to manually register them one by one.

Note that buildProviderModule() only works with providers and controllers that are decorated with the @injectable() decorator.
:::
``````